main.js

```diff
@@ -8,6 +8,37 @@ class Car {
     this.direction = direction
     this.location = location
   }
+  turn() {
+    switch (this.direction) {
+      case 'north':
+        carImage.setAttribute('class', 'north')
+        break
+      case 'south':
+        carImage.setAttribute('class', 'south')
+        break
+      case 'east':
+        carImage.setAttribute('class', 'east')
+        break
+      case 'west':
+        carImage.setAttribute('class', 'west')
+    }
+  }
 }
 
+window.addEventListener('keydown', function (event) {
+  switch (event.key) {
+    case 'ArrowUp':
+      carImage.setAttribute('class', 'north')
```

Comment:
You should `turn` the car and the car's `turn` method should update `this.$img`.